src/data_notification.rs

```diff
@@ -102,4 +102,8 @@ impl DataNotification {
     let (input, notification_body) = Data::parse(input)?;
     Ok((input, Self { long_invoke_id_and_priority, date_time, notification_body }))
   }
+
+  pub fn get_body(&self) -> &Data {
```

Comment:
```suggestion
  pub fn body(&self) -> &Data {
```

Also, move this above the `parse` method.